Readme.md

```diff
@@ -6,23 +6,23 @@
 <!--
 [![Test Coverage][coveralls-image]][coveralls-url]
  -->
-Utility to parse a string bytes (ex: `1TB`) to bytes (`1,000,000,000,000`) and vice-versa.
+Utility to parse a size string in bytes (e.g. `'1kB'`, `'2KiB'`) to numeric (`1000`, `2048`) and vice-versa.
 
-This uses the byte units defined in [ISO/IEC 80000-13:2008][iec], both the binary prefixes and the original SI units.
+This is a fork of the [bytes][bytes] module, except it
 
-This is a fork of the [bytes][bytes] module, except:
+ * supports all of both the binary and decimal prefixes defined by [ISO/IEC 80000-13:2008][iec]
+ * supports [JEDEC][jedec], a compatibility mode resembling the upstream original
+ * uses decimal units by default, which can be overridden per call or by default
 
- * It uses IEC units by default
- * Supports a wider range of units
- * Supports changing to compatability ([JEDEC][jedec]) mode, and formatting in whichever prefix type you prefix (binary, metric, jedec)
-
- TypeScript definitions included.
+TypeScript definitions included.
 
 ## Supported Units
 
-Supported units and abbreviations are as follows and are case-insensitive:
+Supported units are as follows and are case-insensitive. Note that only the abbreviation will be parsed/formatted, the full names are for the reader's understanding only.
```

Comment:
you know it could probably parse the full names too

Reply:
Does it? :o I shall try

Reply:
nah, it doesn't

Reply:
no I mean I could change it fairly easily to support full names